getting-started/data-setup.rst

```diff
@@ -122,7 +122,7 @@ Notes on terminology
 
 First, a coherent set of pixels can have lots of names.
 In this set of tutorials, you will run into three.
-The term exposure, refers to a single image.
+The term exposure refers to a single image.
```

Comment:
Maybe "exposure" - in quotes - and use quotes consistently throughout the next couple of paragraphs to highlight the terms being defined?

Reply:
Good idea. Should be addressed in [425035a](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/425035a870e3602be874460b9bf0c511ba92f8a5), but let me know if not. Thanks...